x-pack/plugins/fleet/server/services/elastic_agent_manifest.ts

```diff
@@ -46,7 +45,10 @@ spec:
               value: "elastic"
             # The basic authentication password used to connect to Elasticsearch
             - name: ES_PASSWORD
-              value: "changeme"
+              value: ""
+            # The Elasticsearch host to communicate with
+            - name: ES_HOST
```

Comment:
just wondering if this variable wasn't skipped on purpose - for standalone mode `ES_HOST` will get rendered to the actual value in configmap

Reply:
https://github.com/elastic/kibana/pull/138393#discussion_r941482408